### PR TITLE
chore(CHANGELOG): update to v1.22.1

### PR DESCRIPTION
**Pull request by smbea**

### CODE REVIEW

**Review by marstamm — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ All notable changes to [bpmn-js-properties-panel](https://github.com/bpmn-io/bpm
 
 ___Note:__ Yet to be released changes appear here._
 
+# 1.22.1
+
+* `FIX`: esure element template properties order is maintained ([#897](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/897))
```
Comment (marstamm) [inline CHANGELOG.md:11]:
```suggestion
* `FIX`: esure element template properties order is maintained ([#897](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/897))
```

**Review by marstamm — COMMENTED** [review]
Does not break but redirects

File: CHANGELOG.md (hunk 1 above)
Comment (smbea) [reply CHANGELOG.md:11]: Nice catch

**Comment by smbea** [comment]
Had to follow up with https://github.com/bpmn-io/bpmn-js-properties-panel/commit/5ea39461a62ea2f9383b776367929636474e117a because I misread the version number

**Pull request merged**